be/src/common/configbase.h

```diff
@@ -160,6 +170,9 @@ class Field {
     void* _storage;
     const char* _defval;
     bool _valmutable;
+    char* _last_set_val;
```

Comment:
Does it has to be c-string? why not std::string?  

if it's std::string, code looks more neat.

Reply:
tks review.
I had updated c style to c++